db/seeds/dev/04_characters.js

```diff
@@ -0,0 +1,49 @@
+const charactersData = require('../../../data/characters');
+
+const createCharacter = async (knex, character) => {
+  const raceID = await getRaceID(knex, character);
+  const classID = await getClassID(knex, character);
+  const weaponID = await getWeaponID(knex, character);
+
+  return knex('characters').insert({
+    name: character.name,
+    race_id: raceID,
+    class_id: classID,
+    weapon_id: weaponID
+  });
+}
```

Comment:
After getting necessary ids they're put to character object and character object is added to DB